security.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
you could add comment in a script or in a README that this file is executed by jenkins (name by convention). Otherwise, from pov of this repo looks like not used.